### ISSUE TITLE
registry_schemas in config file

### ISSUE THREAD

**Issue by jasonmimick** [aws-cloudformation/cfn-python-lint#1835]
Need to add this new parameter to the config file json schema:
`cfnlint/data/CfnLintCli/config/schema.json`
needs
```vim
...
 "ignore_templates": {
  79       "description": "Templates to ignore",
  80       "items": {
  81         "type": "string"
  82       },
  83       "type": "array"
  84     },
  85     "registry_schemas": {
  86       "description": "one or more directories of CloudFormation Registry Resource Schemas",
  87       "items": {
  88         "type": "string"
  89       },
  90       "type": "array"
  91     }
```
```

_Originally posted by @jasonmimick in https://github.com/aws-cloudformation/cfn-python-lint/issues/1732#issuecomment-746845099_

**Comment by PatMyron** [comment]
been meaning to look into generally syncing [CLI/config file arguments](https://github.com/aws-cloudformation/cfn-python-lint#configuration) so they don't need to be maintained separately long term

we can definitely add this in the meantime though

**Comment by jasonmimick** [comment]
Thanks @PatMyron and @kddejong for quick turn around.
I am found this trying to use taskcat, so there is going to be a corresponding issue to add support for this flag there, needs couple changes in `taskcat/_cfn_lint.py` and `cfnlint/helpers/py` , e.g.

pseudo-code
```
def __init__ ...
    if self._cfnlint_config.override_spec:
  48             cfnlint.helpers.override_specs(self._cfnlint_config.override_spec)
```
```
 def registry_schemas(registry_schemas):
 535     """Load the registry schemas"""
 536     for path in registry_schemas:
 537         if path and os.path.isdir(os.path.expanduser(path)):
 538             for f in os.listdir(path):
 539                 with open(os.path.join(path, f)) as schema:
 540                     REGISTRY_SCHEMAS.append(json.load(schema))
```
